### PR TITLE
BTA-1902 added ids to h1s

### PR DESCRIPTION
**Pull request by lmugnai**

### CODE REVIEW

File: app/views/deenrolment/haveYouStoppedSelfEmployment.scala.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
         @components.input_radio(
             field = form("value"),
-            legend = components.heading("haveYouStoppedSelfEmployment.heading").body,
+            legend = components.heading("have-you-stopped-sa", "haveYouStoppedSelfEmployment.heading").body,
```
Comment (liampan) [inline app/views/deenrolment/haveYouStoppedSelfEmployment.scala.html]: is sa self-employment?
Not that it really matters

File: app/views/employer/doesBusinessManagePAYE.scala.html [hunk 2]
```diff
@@ -36,7 +36,7 @@
 
         @components.input_radio(
             field = form("value"),
-            legend = components.heading("doesBusinessManagePAYE.heading").body,
+            legend = components.heading("does-business-manager-paye", "doesBusinessManagePAYE.heading").body,
```
Comment (liampan) [inline app/views/employer/doesBusinessManagePAYE.scala.html]: manager or manage?

File: app/views/deenrolment/haveYouStoppedSelfEmployment.scala.html (hunk 1 above)
Comment (lmugnai) [reply app/views/deenrolment/haveYouStoppedSelfEmployment.scala.html]: well spotted, I'll change it

File: app/views/employer/doesBusinessManagePAYE.scala.html (hunk 2 above)
Comment (lmugnai) [reply app/views/employer/doesBusinessManagePAYE.scala.html]: Manage, I'll change

**Pull request merged**